client/post-editor/editor-post-formats/index.jsx

```diff
@@ -1,69 +1,62 @@
 /**
  * External dependencies
  */
-const React = require( 'react' );
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { map } from 'lodash';
 
 /**
  * Internal dependencies
  */
-const FormRadio = require( 'components/forms/form-radio' ),
-	Gridicon = require( 'components/gridicon' ),
-	PostActions = require( 'lib/posts/actions' ),
-	stats = require( 'lib/posts/stats' ),
-	AccordionSection = require( 'components/accordion/section' );
+import FormRadio from 'components/forms/form-radio';
+import Gridicon from 'components/gridicon';
+import QueryPostFormats from 'components/data/query-post-formats';
+import PostActions from 'lib/posts/actions';
+import { recordStat, recordEvent } from 'lib/posts/stats';
+import AccordionSection from 'components/accordion/section';
 import EditorThemeHelp from 'post-editor/editor-theme-help';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getPostFormats } from 'state/post-formats/selectors';
 
-export default React.createClass( {
+const EditorPostFormats = React.createClass( {
 	displayName: 'EditorPostFormats',
```

Comment:
`displayName` becomes unnecessary when you assign `React.createClass` to a variable (via [`babel-plugin-transform-react-disdplay-name`](https://www.npmjs.com/package/babel-plugin-transform-react-display-name)).